### ISSUE TITLE
Path formatting on Windows

### ISSUE THREAD

**Issue by leonfs** [maxogden/github-oauth#15]
Hi,

When generating the url pathname the library 'path' is being used. As this library has been developed to interact with files it would change the /github/callback pathname to \\github\\calback on windows. 

https://github.com/maxogden/github-oauth/blob/master/index.js#L13

Github expects to the callback URI to be the same as the one specified on OAuth application settings.

```
http://www.myserver.com/github/callback.
```

But the library is sending:

```
http://www.myserver.com/\\github\\callback
```

Forcing Github to reject the authorization request due to MISMATCH URI.

**Comment by maxogden** [comment]
ooh nice find, I guess `path.join` should be replaced with literal `/`

**Comment by max-mapper** [comment]
fixed and published as 0.2.2
